cpu/atmega_common/cpu.c

```diff
@@ -63,12 +63,11 @@ void get_mcusr(void)
 #else
     /* save the reset flags */
 #ifdef MCUCSR
-  mcusr_mirror = MCUCSR;
-  MCUSR = 0;
```

Comment:
Shouldn't this have been
```C
MCUCSR = 0;
```
?

Reply:
Aparantly, Atmel/Microchip renamed the MCUSR register into MCUCSR about the time releasing the ATmega128. I think that avr libc just provides now both names now for all MCUs :-)

In any case, all MCUs supported by RIOT only use the term MCUSR, so that is some piece of cargo cult programming we don't need. (And like won't need in the future, as the avr libc seems to still supply the old name for the newer MCUs for compatibility.)

Reply:
In this case, why not drop the `#ifdef` entirely?

Reply:
Oh you just did :)

Reply:
According to this I got the history incorrect: https://www.ibr.cs.tu-bs.de/projects/mudtn/doxygen/a00726_source.html

It seems that MCUCSR is the older name. I'll just steal the approach they used there to never have to worry about this.